src/libraries/System.IO.FileSystem/src/System/IO/Enumeration/FileSystemEnumerator.cs

```diff
@@ -14,6 +14,8 @@ namespace System.IO.Enumeration
 {
     public unsafe abstract partial class FileSystemEnumerator<TResult> : CriticalFinalizerObject, IEnumerator<TResult>
     {
+        private int _remainingDepth;
```

Comment:
Could we rename this as `_remainingRecursionDepth`?

Reply:
Done.